python/dolfinx/wrappers/fem.cpp

```diff
@@ -457,12 +457,14 @@ void fem(py::module& m)
       "locate_dofs_topological",
       [](const std::vector<
              std::reference_wrapper<const dolfinx::fem::FunctionSpace>>& V,
-         const int dim, const Eigen::Ref<const Eigen::ArrayXi>& entities,
+         const int dim, const py::array_t<std::int32_t>& entities,
          bool remote) -> std::array<py::array, 2> {
         if (V.size() != 2)
           throw std::runtime_error("Expected two function spaces.");
+        auto sp
+            = tcb::span<const std::int32_t>(entities.data(), entities.size());
         std::array<std::vector<std::int32_t>, 2> dofs
-            = dolfinx::fem::locate_dofs_topological({V[0], V[1]}, dim, entities,
+            = dolfinx::fem::locate_dofs_topological({V[0], V[1]}, dim, sp,
```

Comment:
Replace `sp` with `tcb::span(entities.data(), entities.size())`

Reply:
That's how I first tried it, but it complained that

```
fem.cpp:467:15: error: no matching function for call to 'locate_dofs_topological'
DirichletBC.h:53:42: note: candidate function not viable: expects an l-value for 3rd argument
```

Reply:
There is a missing `const` - I just posted a comment.